deposit_contract/contracts/validator_registration.v.py

```diff
@@ -1,8 +1,7 @@
 MIN_DEPOSIT_AMOUNT: constant(uint256) = 1000000000  # Gwei
 MAX_DEPOSIT_AMOUNT: constant(uint256) = 32000000000  # Gwei
-GWEI_PER_ETH: constant(uint256) = 1000000000  # 10**9
 CHAIN_START_FULL_DEPOSIT_THRESHOLD: constant(uint256) = 16384  # 2**14
-DEPOSIT_CONTRACT_TREE_DEPTH: constant(uint256) = 32
+DEPOSIT_TREE_DEPTH: constant(uint256) = 32
```

Comment:
Sorry `DEPOSIT_TREE_DEPTH` was left out in the progressive merkle tree update. Can you replace `32` constants in the for loops with `DEPOSIT_TREE_DEPTH`?
And also remove `TWO_TO_POWER_OF_TREE_DEPTH`.

Reply:
Done !